packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/datetime/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+---
+
+In this package, there are two components (`DateTime` and `RelativeDateTime`) that will return a date following [Forma 36 guidelines](https://f36.contentful.com/guidelines/date-and-time/).
+
+## DateTime
+
+The DateTime component will format any date to a human friendly format and wrap it in a `<time>` tag.
+It allows four different forms of display that can be controlled by changing the `format` prop:
+
+| `format` option | Result                 |
+| --------------- | ---------------------- |
+| `full`          | 12 Aug 2020 at 8:00 AM |
+| `day`           | 12 Aug 2020            |
+| `time`          | 8:00 AM                |
+| `weekday`       | Wed, 12 Aug            |
+
+### Examples
+
+```tsx static=true
+import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<DateTime date={new Date()} />; // passing a JS Date
+<DateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+All the usages above will result in 'Tue, 17 Aug 2021 at 3:45 PM'
+
+## RelativeDateTime
+
+Provides a relative date string for resolutions ranging from seconds up through years. Should be used when absolute date and time is unnecessarily specific.
+
+### Examples
+
+If we consider today as August 18th of 2021 and we use pass the following values to the Component
+
+```tsx static=true
```

Comment:
Do we want to only show the code here? Or should we also show the rendered result?

Reply:
🤔 

I guess the rendered result as well

I was confused about the whole `static=true` thing
tbh, when do we want to show only the code?

Reply:
I believe that in most of the cases we will want to show the rendered component, we may only want to show the code when it are only snippets that don't necessary render a component.